rke/templates/templates.go

```diff
@@ -68,7 +68,7 @@ const (
 	nginxIngressv18    = "nginxingress-v1.8"
 	nginxIngressV115   = "nginxingress-v1.15"
 	nginxIngressV11512 = "nginxingress-v1.15.12"
-	nginxIngressv046   = "nginxingress-v0.46.0"
+	nginxIngressv0481  = "nginxingress-v0.48.1"
```

Comment:
Is no released version using 046?

Reply:
No, the latest released uses `nginx-0.43.0-rancher3`